iree/samples/simple_embedding/BUILD

```diff
@@ -14,25 +14,53 @@
 
 load("//iree:build_defs.oss.bzl", "iree_cmake_extra_content")
 load("//iree/tools:compilation.bzl", "iree_bytecode_module")
+load("//build_tools/embed_data:build_defs.bzl", "c_embed_data")
 
 package(
     default_visibility = ["//visibility:public"],
     features = ["layering_check"],
     licenses = ["notice"],  # Apache 2.0
 )
 
+c_embed_data(
+    name = "simple_embedding_test_llvm_aot_rv64",
+    srcs = [
+        "data/simple_embedding_test-llvm-aot_rv64.vmfb",
```

Comment:
Can you add a comment for how to regenerate this `.vmfb` file?

Reply:
Done.